test/Constraints/diagnostics.swift

```diff
@@ -1117,3 +1117,10 @@ func badTypes() {
   // expected-error@-1 {{type of expression is ambiguous without more context}}
   // FIXME: terrible diagnostic
 }
+
+// rdar://problem/35198459 - source-compat-suite failure: Moya (toType->hasUnresolvedType() && "Should have handled this above")
```

Comment:
This should go into test/Compatibility/tuple_arguments and test/Constraints/tuple_arguments

Reply:
Thanks, will move! I can't ever seems to figure out where to put new tests :(

Reply:
That's OK, they're not well organized to begin with IMO and everyone has different opinions, but I've been trying to keep all SE-0110-related test cases in one place.